docs/docs/index.md

```diff
@@ -0,0 +1,13 @@
+# Mechaphlowers
+
+## Try some maths
+
+$$
+\sum_{i=0}^n \pi^2 = \Delta\frac{(n^2+\epsilon)(2n+1)}{6\phi}
+$$
+
+## Try to include some plotly figure
+
+
+{ % include_relative test_figure_potly.html % }
```

Comment:
This doesn't work on my machine: 
If I navigate to the served documentation in a browser I only see the code above, not the figure itself (but maybe we can leave the setup for plotly figures for another PR? This one is already quite big)

![image](https://github.com/user-attachments/assets/27cb2971-a864-4f6b-bca5-2e8fd58d1a1e)

Reply:
After talk : ok for moving it in #9